<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GhostVault</class>
 <widget class="QWidget" name="GhostVault">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ghost Vault Balance: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0 Ghosted NIX</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="unconfirmed_label">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0 Unconfirmed </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>138</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Ghost NIX to Vault</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>801</width>
           <height>96</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="ghostAmount">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>e.g. 17</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ghostNIXButton">
              <property name="toolTip">
               <string>Ghost NIX</string>
              </property>
              <property name="text">
               <string>&amp;Ghost NIX</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../res/icons/add.png</normaloff>../res/icons/add.png</iconset>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Un-Ghost NIX from Vault</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>812</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Un-Ghost To:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="convertGhostButton">
         <property name="geometry">
          <rect>
           <x>455</x>
           <y>70</y>
           <width>181</width>
           <height>25</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Convert Ghosted NIX</string>
         </property>
         <property name="text">
          <string>&amp;Un-Ghost NIX</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../res/icons/edit.png</normaloff>../res/icons/edit.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="convertGhostToThirdPartyAddress">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>18</y>
           <width>350</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>e.g. NGshe11sePfsXGNn6oJQ1CzeGMGhhYTQxV</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="convertGhostToMeCheckBox">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>20</y>
           <width>150</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Un-Ghost to myself</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="convertNIXAmount">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>e.g. 17</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>53</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Amount:</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Transfer from Vault to Vault</string>
        </attribute>
        <widget class="QPushButton" name="transferGhostButton">
         <property name="geometry">
          <rect>
           <x>455</x>
           <y>70</y>
           <width>181</width>
           <height>25</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Convert Ghosted NIX</string>
         </property>
         <property name="text">
          <string>&amp;Transfer</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../res/icons/edit.png</normaloff>../res/icons/edit.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="transferNIXAmount">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>e.g. 17</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="transferGhostTo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>18</y>
           <width>350</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>e.g. yd12XEHUs1y2Z9NYSufNxKOWftA74gYb8Q7l4jrGlGRpc6/ry0X4FUfApaFkJtyAYAAXJKWSpw3jshPeKQ9isJ6ysSIZ3BDf8qEyDl5iS9WGey1dIR6wOyhm2njcVzL6pltNS32cWDYuFuhtG8opsXmh0v1B12h7IcME/28QxIMA</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>53</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Amount:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>812</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Transfer To:</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
